docs/guides/src/main/operator/installation.adoc

```diff
@@ -5,14 +5,48 @@
 
 <@tmpl.guide
 title="Keycloak Operator Installation"
-summary="How to install the Keycloak Operator on Kubernetes">
-
-TODO: https://github.com/keycloak/keycloak/issues/10786
+summary="How to install the Keycloak Operator on Kubernetes and OpenShift">
 
 == Keycloak Operator Installation
+In this guide we will show how to install the Keycloak Operator in your Kubernetes or OpenShift cluster.
+
+=== OLM Installation
+
+You can install the Keycloak Operator in Kubernetes environment is to use Operator Lifecycle Manager (OLM).
```

Comment:
```suggestion
The recommended way to install the Keycloak Operator in Kubernetes environments is to use the Operator Lifecycle Manager (OLM).
```

Reply:
plus perhaps a Link to OLM page.

Reply:
Fixed in a different way and added the link